### DISCUSSION TITLE
github actions are broken on `linux` env.

### DISCUSSION THREAD

**Discussion by siail** [cookiecutter/cookiecutter-django#3656] [General]
Apparently, GitHub actions are broken currently because of `black` or its dependency `click` throwing errors.

On windows, `pre-commit` is working fine.
On macOS, `pre-commit` is working fine as well.
But on Linux, (I have tested both on local ~> `ubuntu20.04` and the `test` job is also not passing (`ci.yml`). Along with that, tests running inside the docker containers are also failing.

If anyone knows the root cause or any temporary fix. Please mention.

Thanks

**Comment by browniebroke** [comment]
See https://github.com/psf/black/pull/2966

It should be fixed in black version 22.3

**Reply by siail** [reply]
✅ Yeah, its passing now. At least on my local machine.

so are you planning to update the black version in `./pre-commit-config.yml` and `./requirements.txt`? I can do it.

**Reply by browniebroke** [reply]
We should receive some automated updates in a little while, but if you want to go ahead, it might save some trouble to others.

**Reply by siail** [reply]
Yeah, I am locally running all the tests to make sure things are working. I was figuring that out for the past 2 hours. The tests were failing on my fork but when I looked at the main project, the tests were passing. I thought I messed up something but it wasn't the case.
To prevent trouble for others. I want to update it as soon as possible.

**Reply by siail** [reply]
#3657
